pandas/tests/io/excel/test_openpyxl.py

```diff
@@ -85,30 +85,46 @@ def test_write_cells_merge_styled(ext):
         assert xcell_a2.font == openpyxl_sty_merged
 
 
-@pytest.mark.parametrize("write_only", [True, False])
-def test_kwargs(ext, write_only):
+@pytest.mark.parametrize("iso_dates", [True, False])
+def test_kwargs(ext, iso_dates):
     # GH 42286
     # openpyxl doesn't utilize kwargs, only test that supplying a kwarg works
```

Comment:
Looks like this comment is (no longer?) true, is that right? If so, can you remove.